src/renderer/components/watch-video-chapters/watch-video-chapters.vue

```diff
@@ -1,5 +1,12 @@
 <template>
   <ft-card class="videoChapters">
+    <div
+      v-observe-visibility="observeVisibilityOptions"
+    >
+      <!--
+        Dummy element to be observed by Intersection Observer
+      -->
+    </div>
```

Comment:
Instead of creating a new element, you can probably just add the directive to the `<div>` with `v-show="showChapters"` on it, that way it won't fire when the chapters aren't visible and you won't need the dummy element.

Reply:
Done~